models.rdf

```diff
@@ -66,6 +66,14 @@
       <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
     </rdfs:Class>
 
+    <rdfs:Class rdf:about="http://pcdm.org/models#FileSet">
```

Comment:
Also digitization (or capture) date/event information, which I'd also say falls mostly under technical. But I agree, we need to scope out Fileset as a class a bit more. (Sorry to jump in)